modules/test/playwright/tests/commerce/utils/commerce.ts

```diff
@@ -21,8 +21,8 @@ export async function classicCommerceSetUp(
 	return initializerSetUp(
 		apiHelpers,
 		'com.liferay.commerce.site.initializer',
-		'Commerce Classic',
-		'Liferay Commerce Channel',
+		siteName,
+		siteName,
```

Comment:
These 2 could be null as they will anyway fallback in the function body, but just to be explicit...